src/components/MainPage.jsx

```diff
@@ -0,0 +1,47 @@
+import { useEffect, useState } from 'react'
+import ArticleCard from './ArticleCard';
+import Pagination from './Pagination';
+
+
+export default function MainPage() {
+	const [articles, setArticles] = useState([]);
+	const [currentPage, setCurrentPage] = useState(1);
+	const articlesPerPage = 10;
+
+	useEffect(() => {
```

Comment:
You should have some sort of loading state and a way of indication that to the user

Reply:
Be aware that fetch will not give errors for 400s or 500s, you may want to switch to axios

Reply:
done